paper-input-container.html

```diff
@@ -90,6 +90,8 @@
 `--paper-input-container-label-focus` | Mixin applied to the label when the input is focused | `{}`
 `--paper-input-container-label-floating` | Mixin applied to the label when floating | `{}`
 `--paper-input-container-input` | Mixin applied to the input | `{}`
+`--paper-input-container-input-focus` | Mixin applied to the input | `{}`
```

Comment:
Please change this comment to 'Mixin applied to the input when focused', like the `-label-focus` mixin is commented. Similar comment for the `-input-invalid` one